### ISSUE TITLE
[UI] Second time the option from scrollable list in "add robot" tab is clicked, the entry below the one clicked is chosen 

### ISSUE THREAD

**Issue by xalpol12** [xalpol12/kuka-ar-all#11]
Also after clicking on the category once again, two options are highlighted in blue.

[Link to a video](https://drive.google.com/file/d/1Rxn_fFAdD1QEKS-zrnamK0OqlIam4UtY/view?usp=sharing)

**Comment by CoyotWilly** [comment]
It's so funny as those were the problems that I've solved this morning. It's so annoying that depending on Unity run you are seeing different behaviors

**Comment by xalpol12** [comment]
Ok, I will try restarting Unity then

**Comment by CoyotWilly** [comment]
Both options are possible the bug with window not docking at the finish position it's sth that will require some bigger research as there is only simple IF statement. However select issue it's well know to me, I will stop myself from posting new pull request as long as I'm not done with all UI features. 

Maybe this will save as both some stress and time

**Comment by xalpol12** [comment]
Ok, so let me merge #10 to clean /UI branch (as it's really hard to see the changes now) and we will keep in mind the issues.

**Comment by CoyotWilly** [comment]
Ok, here is my strategy for this Unity related bull shit:
- new branch from New Main
- current progress from already existing feature/more-options branch will me moved
- this branch will stay until we done with UI and it's backend connection 
- I'm gonna try to avoid messing up with view as much as I was doing so far 
- some issues will stay with use no matter what as I don't see any patter in their behavior

**Comment by xalpol12** [comment]
Yeah, creating new branch from main would be the best option. It is weird that Unity creates that many issues with merging and rebasing - especially because no one else is touching UI related files.
